### ISSUE TITLE
Viki: Download subtitles - HTTP Error 422: Unprocessable Entity

### ISSUE THREAD

**Issue by donnekgit** [ytdl-org/youtube-dl#29891]
- [ x] I'm reporting a broken site support
- [ x] I've verified that I'm running youtube-dl version **2021.06.06**
- [ x] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x ] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--write-sub', '--sub-format', 'srt', '--sub-lang', 'ko', '--skip-download', '--verbose', 'https://www.viki.com/videos/1115053v']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.06.06
[debug] Python version 3.8.10 (CPython) - Linux-5.11.0-27-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Proxy map: {}
[viki] 1115053v: Downloading video JSON
ERROR: Unable to download JSON metadata: HTTP Error 422: Unprocessable Entity (caused by <HTTPError 422: 'Unprocessable Entity'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 634, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
```

## Description

Trying to download the srt Korean subtitles for an episode in a Korean drama on Viki.

Getting the above "HTTP Error 422: Unprocessable Entity" error.

This seems to be different from #29224.

**Comment by kikuyan** [comment]
#29502, #29459

**Comment by Footur** [comment]
FYI: Viki is working with [yt-dlp](https://github.com/yt-dlp/yt-dlp).

**Comment by donnekgit** [comment]
Yes, that's what I ended up using.

**Comment by dirkf** [comment]
Fixed in #30596: `Guardian - The Lonely and Great God - Episode 1-1115053v.ko.srt`
